src/python/pants/backend/graph_info/tasks/pathdeps.py

```diff
@@ -1,17 +1,25 @@
 # coding=utf-8
-# Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
                         unicode_literals, with_statement)
 
+import os
+
+from pants.base.build_environment import get_buildroot
+from pants.base.deprecated import deprecated_conditional
 from pants.task.console_task import ConsoleTask
 
 
 class PathDeps(ConsoleTask):
   """List all paths containing BUILD files the target depends on."""
 
   def console_output(self, targets):
+    deprecated_conditional(lambda: True, '1.5.0.dev0',
+                           'The `pathdeps` goal is deprecated. Please use `filedeps` instead.')
     def is_safe(t):
-      return hasattr(t, 'address') and hasattr(t.address, 'build_file')
-    return set(t.address.build_file.parent_path for t in targets if is_safe(t))
+      return hasattr(t, 'address') and hasattr(t.address, 'rel_path')
```

Comment:
nit: I remember in one of my previous reviews, you suggest using "getattr" instead of "hasattr". I don't quite remember the reasoning behind it, but just wonder if it applies here.